ebml/EbmlElement.h

```diff
@@ -665,7 +665,7 @@ class EBML_DLL_API EbmlElementDefaultSameStorage : public EbmlElementDefault<T>
     explicit operator T() const { return Value; }
 
   private:
-    T Value;
+    T Value{};
```

Comment:
While this is correct, it seems odd to intialize to a value that is not the default value.

Maybe we should have constructors of EbmlElementDefaultSameStorage when the class has a default value (which will initialize to the proper default value) and one with the default value for the type.

Reply:
How is this not default?

I'd rather have it like this with a using declaration instead of defining constructors.